Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3
+
+WORKDIR .
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
```

Comment:
Since this copies over everything starting at the root of the repo, it'd be better to separate only what's needed in the container and copy that over.